include/drivers/sensor.h

```diff
@@ -90,6 +90,8 @@ enum sensor_channel {
 	SENSOR_CHAN_HUMIDITY,
 	/** Illuminance in visible spectrum, in lux. */
 	SENSOR_CHAN_LIGHT,
+	/** Ultraviolet index, in UVI. */
+	SENSOR_CHAN_UV,
```

Comment:
I wouldn't put this in the middle of the other light channels given that the units are not the same.
Its also not obvious that `SENSOR_CHAN_UV` and `SENSOR_CHAN_IV` are returning very different things.
Perhaps `SENSOR_CHAN_UV_INDEX`?